.github/workflows/add-to-triage.yml

```
@@ -0,0 +1,16 @@
+name: Add to Triage
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@RELEASE_VERSION
```

```suggestion
      - uses: actions/add-to-project@v0.4
```

---

I just tried this in a test repo and got `` Error: Unable to resolve action `actions/add-to-project@v0.4`, unable to find version `v0.4` ``. Not sure what's wrong with `v0.4`, but `v0.4.0` worked well:

```suggestion
      - uses: actions/add-to-project@v0.4.0
```